[2025-08-25T13:49:17.643+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mamda_monthly_etl_pipeline.cleanup_temp_files manual__2025-08-25T13:49:12.439779+00:00 [queued]>
[2025-08-25T13:49:17.651+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mamda_monthly_etl_pipeline.cleanup_temp_files manual__2025-08-25T13:49:12.439779+00:00 [queued]>
[2025-08-25T13:49:17.652+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-08-25T13:49:17.665+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): cleanup_temp_files> on 2025-08-25 13:49:12.439779+00:00
[2025-08-25T13:49:17.671+0000] {standard_task_runner.py:60} INFO - Started process 683 to run task
[2025-08-25T13:49:17.673+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mamda_monthly_etl_pipeline', 'cleanup_temp_files', 'manual__2025-08-25T13:49:12.439779+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/mamda_monthly_etl.py', '--cfg-path', '/tmp/tmply3sduqt']
[2025-08-25T13:49:17.675+0000] {standard_task_runner.py:88} INFO - Job 81: Subtask cleanup_temp_files
[2025-08-25T13:49:17.723+0000] {task_command.py:423} INFO - Running <TaskInstance: mamda_monthly_etl_pipeline.cleanup_temp_files manual__2025-08-25T13:49:12.439779+00:00 [running]> on host 9f5b6f3413d6
[2025-08-25T13:49:17.796+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mamda_etl' AIRFLOW_CTX_DAG_ID='mamda_monthly_etl_pipeline' AIRFLOW_CTX_TASK_ID='cleanup_temp_files' AIRFLOW_CTX_EXECUTION_DATE='2025-08-25T13:49:12.439779+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-25T13:49:12.439779+00:00'
[2025-08-25T13:49:17.798+0000] {mamda_monthly_etl.py:198} INFO - Cleanup finished
[2025-08-25T13:49:17.798+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-08-25T13:49:17.808+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=mamda_monthly_etl_pipeline, task_id=cleanup_temp_files, execution_date=20250825T134912, start_date=20250825T134917, end_date=20250825T134917
[2025-08-25T13:49:17.847+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-25T13:49:17.865+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
